test/performance/shmem_perf_suite/common.h

```diff
@@ -172,10 +176,16 @@ long bar_psync[SHMEM_BARRIER_SYNC_SIZE];
 /* default settings with no input provided */
 static inline 
 void set_metric_defaults(perf_metrics_t *metric_info) {
+    char *val = NULL;
+    metric_info->trials_multiplier = 1.0; /* Default 1 */
+    val = getenv("SOS_TRIALS_MULTIPLIER");
```

Comment:
Should we use `SHMEM_PERF_SUITE_TRIALS_MULTIPLIER`? Actually, I will prefer adding this as an argument flag rather than an env variable? Thoughts?

Reply:
Yeah, that would definitely be more consistent with the existing tool...

Would that work on your end @minsii?  While the env var is slightly easier b/c you just set it once, we really have no precedent for such an env var in shmem_perf_suite, and adding a new flag may be generally more useful.